pkg/minikube/assets/vm_assets.go

```diff
@@ -117,6 +119,7 @@ func NewFileAsset(src, targetDir, targetName, permissions string) (*FileAsset, e
 			Permissions: permissions,
 		},
 		reader: io.NewSectionReader(f, 0, info.Size()),
+		file:   f,
```

Comment:
if in `info, err := os.Stat(src)` `err` is not `nil`, `f` is not closed

Reply:
Change order for os.Stat and os.Open